### PR TITLE
Raindrops fix

### PR DESCRIPTION
**Pull request by axtens**
@Caraciola 's fix

### CODE REVIEW

File: exercises/practice/raindrops/.meta/config.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "authors": [
-    "axtens"
+    "Caraciola"
```
Comment (ErikSchierboom) [inline exercises/practice/raindrops/.meta/config.json:3]: You're still an author, but you can have multiple:

```suggestion
    "axtens",
    "Caraciola"
```

**Comment by axtens** [comment]
This need for a zip file password really is an issue. Oh well. Meanwhile, the raindrops fix will make the tests work

**Comment by axtens** [comment]
Seeing as I know what the token should contain, how do I add it so that the script will work?

File: bin/test [hunk 2]
```diff
@@ -55,6 +55,8 @@ for concept_exercise_dir in ./exercises/concept/*/; do
     fi
 done
 
+EXIT_CODE=0
+
```
Comment (ErikSchierboom) [inline bin/test:59]: Why was this added? I'm gonna remove this.

Comment (ErikSchierboom) [inline bin/test:59]:
```suggestion
```

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
@axtens This is weird. We'll look into it
